Project-II-Calculator/calculator/src/components/DisplayComponents/Display.css

```diff
@@ -17,4 +17,11 @@
 .calc-body {
   display: flex;
   width: 100%;
-}
\ No newline at end of file
+}
+
+@media all and (max-width: 500px) {
+  .display {
+    font-size: 3rem;
+  }
+}
```

Comment:
**Day 2 Review**

- Great job tackling the stretch goal on calculator Chinedu. You managed to implement functionality  and get a result displayed 👍  there is no more until you have a really good functionality going on there 🎖 
- Your code is clear and i like how You are using .map() generate new components and pass props to them. This is how You create reusable components.
- Overall great job 👍